tests/Fixtures/FixedCookieHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use WMDE\Fundraising\Frontend\Infrastructure\CookieHandler;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class FixedCookieHandler implements CookieHandler {
```

Comment:
This class confuses me a bit because from the name I expected it to always return the same value for the cookies, some kind of "immutable cookie jar". Instead, it ignores the values that are set and instead always returns the fixed values. If you further use this class (see my other comments on usage), could you please mention how this class works in a doc comment?